### DISCUSSION TITLE
Last version(1.4.36) broke SQLModel relationship. Any idea why ?

### DISCUSSION THREAD

**Discussion by jossefaz** [sqlalchemy/sqlalchemy#7972] [Usage Questions]
From the last update, SQLModel relationships declarations are not working anymore and not recognized as a part of the call attributes.
This issue was open in SQLModel repository : https://github.com/tiangolo/sqlmodel/issues/315

I am trying to understand from the release note what could be the breaking change, but I don't even know from where to start.... Does anyone have an idea where to precisely looking?

**Comment by jossefaz** [comment]
In SQLModel the DeclarativeMeta is using the `__init__` method. 
https://github.com/tiangolo/sqlmodel/blob/8d1b6f079adad47cc242710f6cb1790a8ad8fbd2/sqlmodel/main.py#L372
Could it be the issue ? Since this was change here : 
https://github.com/sqlalchemy/sqlalchemy/releases/tag/rel_1_4_36
https://github.com/sqlalchemy/sqlalchemy/issues/7900

**Reply by zzzeek** [reply]
it is absolutely the change in DeclarativeMeta, so SQLModel should update their metaclass to populate cls *and* dict with their attributes, that way the code will work with either metaclass approach.

**Reply by jossefaz** [reply]
Thanks

**Reply by tiangolo** [reply]
Thanks! :bow:
